docs/concepts.md

```diff
@@ -4,39 +4,34 @@ title: Concepts
 sidebar_label: Concepts
 ---
 
-## Specification concepts
-
-Specification uses following vocabulary to define GraphQL queries
```

Comment:
When I originally read this sentence, I thought we were missing some list here that was supposed to follow. It didn't dawn on me until later that the "following vocabulary" referred to the subsequent sections.

I think we could drop this section entirely. At the very least, the header for the section should be bigger than the rest to indicate the other sections are actually sub-sections of this one. But we don't use that sort of formatting anywhere else in the document.

Reply:
We put some questionable concepts into spec. They need more information to stay there and we hoping to extend it. 

For the moment I got some initial review from couple people and I see that we need to differentiate between:

- Capabilities (like pagination, portability etc.)
- Variations of the spec (different variants of capabilities)
- Core use case. 

We have done bad job on making this distinction.
If we look into other emerging standards like graphql HTTP:
https://github.com/APIs-guru/graphql-over-http 

There are so well written and covering many advanced topics etc.